db/models/Comment.js

```diff
@@ -9,11 +9,11 @@ module.exports = (sequelize, DataTypes) => {
   Comment.associate = (models) => {
     Comment.belongsTo(models.User, {
       foreignKey: 'userId',
-      as: 'user'
+      as: 'author'
     });
     Comment.belongsTo(models.Article, {
       foreignKey: 'articleId',
-      as: 'article',
+      as: 'comment',
```

Comment:
kindly revert this back to what it was which was `article`. It is the model that is being associated with the comment model that is  to be set as value to `as`
i.e
it should be `as: 'article'` and not `as: 'comment'`

Reply:
Good catch. I will look into this immediately,